plugin/wavesurfer.spectrogram.js

```diff
@@ -230,7 +247,184 @@ WaveSurfer.Spectrogram = {
         }
 
         return newMatrix;
-    }
+    },
+
 };
 
 WaveSurfer.util.extend(WaveSurfer.Spectrogram, WaveSurfer.Observer);
+
+function FFT(bufferSize, sampleRate, windowFunc, alpha) {
```

Comment:
Could you also add a comment about the code being based on https://github.com/corbanbrook/dsp.js?